ali-dbhub-client/package.json

```diff
@@ -43,8 +44,8 @@
     "monaco-editor-nls": "^3.0.0",
     "monaco-editor-webpack-plugin": "^7.0.1",
     "npm": "^9.6.2",
-    "react": "17.x",
-    "react-dom": "17.x",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
```

Comment:
For PRs that modify documentation, dependency version numbers should not be changed. If a change in dependency version number is required, a separate PR should be opened for it and tests need to be conducted.